portal/src/graphql/adminapi/UserDetailsConnectedIdentities.module.css

```diff
@@ -61,6 +61,7 @@
 .cellName {
   grid-area: name;
   font-weight: 600;
+  font-size: 14px;
```

Comment:
To change font-size, render a Text inside a Link and set the variant prop of Text.